ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ModifyVulnReport/CloneVulnReportPage.tsx

```diff
@@ -66,6 +66,10 @@ function CloneVulnReportPage() {
         createReport(formValues);
     }
 
+    function onClose() {
+        history.push(vulnerabilityReportsPath);
```

Comment:
I wanted to explain a few more considerations I had about the flow for `onClose`:

<img width="246" alt="Screenshot 2023-08-15 at 1 30 49 PM" src="https://github.com/stackrox/stackrox/assets/4805485/05e94c02-a88c-4e00-b257-606b096151ea">

1. When the user clicks on `Cancel`, it goes back to the reports table
2. When the user clicks on `Cancel`, the user gets a confirmation dialog that asks if they are sure they want to cancel the report creation process. If they are sure, they go back to the reports table
3. We don't even have a `Cancel` button. The user can just use the breadcrumbs to go back to the reports table

Reply:
cc: @zhenpesky

Reply:
@sachaudh  If you are asking about the general user experience, options 2 and 3 would be the ideal solution to always trigger to confirmation dialog when a user has inputted their information in the form. 
If the form is empty numbers 1 and 3 should take users directly back to the table without the need to confirm anything since users didn't provide any inputs.

Reply:
@zhenpesky Added a confirmation modal for canceling 

<img width="719" alt="Screenshot 2023-08-15 at 7 43 31 PM" src="https://github.com/stackrox/stackrox/assets/4805485/0a47ab9b-ad5f-413a-afdf-195e35b38c46">

Reply:
@sachaudh just keep it short and precise of the result in this sentence "All unsaved changes will be lost." this should be enough as it is a non-destructive action. V.S. A deleting action of a report would need to be more descriptive because it is permanent.